app/services/feature_flag.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class FeatureFlag
+  # Long-lived settings that are often environment-specific
+  PERMANENT_SETTINGS = %i[
+  ].freeze
```

Comment:
Is this something that would benefit from a permanent, env specific feature flag? https://github.com/DFE-Digital/early-careers-framework/blob/84270a33adbb917bbafbd0c31307a323c5778ebf/app/controllers/users/sessions_controller.rb#L9

Reply:
This service gives us the flexibility to define the state of each flag outside of code, per environment. In my mind, feature flags should care about the state, not the environment.